src/commands/Announcement/announcement.ts

```diff
@@ -3,42 +3,44 @@ import { LanguageKeys } from '#lib/i18n/languageKeys';
 import { SkyraCommand } from '#lib/structures';
 import type { GuildMessage } from '#lib/types';
 import { Events, PermissionLevels } from '#lib/types/Enums';
-import { map } from '#utils/common';
+import { map, seconds } from '#utils/common';
 import { BrandingColors } from '#utils/constants';
-import { canSendMessages, promptConfirmation } from '#utils/functions';
+import { promptConfirmation } from '#utils/functions';
 import { announcementCheck, DetailedMentionExtractionResult, extractDetailedMentions } from '#utils/util';
 import { ApplyOptions } from '@sapphire/decorators';
-import { RESTJSONErrorCodes } from 'discord-api-types/v6';
-import { DiscordAPIError, MessageEmbed, Role, TextChannel } from 'discord.js';
+import { canSendMessages, GuildTextBasedChannelTypes } from '@sapphire/discord.js-utilities';
+import { send } from '@skyra/editable-commands';
+import { RESTJSONErrorCodes } from 'discord-api-types/v9';
+import { DiscordAPIError, MessageEmbed, MessageOptions, Role } from 'discord.js';
 import type { TFunction } from 'i18next';
 
 const flags = ['excludeMentions', 'em'];
 const empty: DetailedMentionExtractionResult = { channels: new Set(), roles: new Set(), users: new Set() };
 
 @ApplyOptions<SkyraCommand.Options>({
 	aliases: ['announce'],
-	bucket: 6,
-	cooldown: 30,
+	cooldownLimit: 6,
+	cooldownDelay: seconds(30),
```

Comment:
Just making sure, is this meant to be ms?

Reply:
Already answered above